install.ps1

```diff
@@ -1,88 +1,66 @@
+#!/usr/bin/env pwsh
 # Copyright 2018 the Deno authors. All rights reserved. MIT license.
 # TODO(everyone): Keep this script simple and easily auditable.
+
 param (
-  [alias("v")]
-  [string] $version
+  [ValidatePattern('^v(\d+).(\d+).(\d+)$')]
+  [String] $Version = 'v0.2.6'
 )
 
-$ErrorActionPreference = "Stop"
-
-# Enable TLS 1.2 since it is required for connections to GitHub.
-[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+$ErrorActionPreference = 'Stop'
 
-# Helper functions for pretty terminal output.
-function Write-Part ([string] $Text) {
-  Write-Host $Text -NoNewline
-}
-function Write-Emphasized ([string] $Text) {
-  Write-Host $Text -NoNewLine -ForegroundColor "Yellow"
-}
-function Write-Done {
-  Write-Host " done" -NoNewline -ForegroundColor "Green";
-  Write-Host "."
+if ($PSVersionTable.PSVersion.Major -lt 6) {
+  $IsWindows = $true
```

Comment:
Are $IsWindows and $IsMacOS powershell builtins? 
Otherwise I don't understand this logic -- I'm pretty sure it's possible to have PS 6 on windows.

Reply:
Yes, `$IsWindows`, `$IsMacOS` and `$IsLinux` are built-ins in PowerShell Core (starting from PowerShell v6). So for <v6, I'm adding the built-ins.